relay-server/src/services/processor.rs

```diff
@@ -1400,20 +1400,20 @@ impl EnvelopeProcessorService {
         project_id: ProjectId,
         project_info: Arc<ProjectInfo>,
         mut event_fully_normalized: EventFullyNormalized,
-    ) -> Result<Option<EventFullyNormalized>, ProcessingError> {
+    ) -> Result<EventFullyNormalized, ProcessingError> {
```

Comment:
Just refactored this to do like I specified above, that is, just returning the non-modified value in case it was not modified. It makes the caller side cleaner.